### PR TITLE
feat(project): Add fields description

### PR DESCRIPTION
**Pull request by jeremyvergnas**
feat(project): Add fields description

### CODE REVIEW

**Review by bas080 — APPROVED** [review]

File: src/component/framework/bootstrap/form-group.html [hunk 1]
```diff
@@ -6,11 +6,19 @@
     <label
       for="${element.id}"
       t.bind="labelText"
-      if.bind="showsLabel">${labelText}</label>
+      if.bind="showsLabel">
+      ${labelText}
+    </label>
     <slot></slot>
+    <small
```
Comment (doktordirk) [inline src/component/framework/bootstrap/form-group.html]: should be generic span not small i.d say

**Review by doktordirk — COMMENTED** [review]
why is there description and help needed. that seems to cover special needs instead of being generic. 

ps: would be nice if any of this would be documented

**Comment by jeremyvergnas** [comment]
@doktordirk `messages` would be used for validation purpose (error/warning messages), and `descriptions` would be the actual descriptions of the fields.

**Comment by doktordirk** [comment]
that sounds reasonable

**Pull request merged**